packages/site/src/layout/NavMenu.tsx

```diff
@@ -13,22 +13,11 @@ export const NavMenu = () => {
   const [open, setOpen] = useState(false);
 
   return (
-    <Box
-      width={200}
-      background="surface--background"
-      border={{ right: "base" }}
```

Comment:
This border was bleeding into ALL the borders so I deleted it altogether
![image](https://github.com/user-attachments/assets/ed14e7de-f652-4bed-91ea-ebd858473f19)